test/cut/test_feature_extraction.py

```diff
@@ -0,0 +1,44 @@
+from tempfile import TemporaryDirectory
+from unittest.mock import Mock
+
+import pytest
+import numpy as np
+
+from lhotse import Recording, Cut, Fbank
+
+
+@pytest.fixture
+def audio_source():
+    # Return a mocked "AudioSource" object that loads a 1s long 1000Hz sine wave
+    source = Mock()
+    source.load_audio = Mock(return_value=np.sin(2 * np.pi * 1000 * np.arange(0, 8000, dtype=np.float32)))
+    source.channel_ids = [0]
+    return source
+
+
+@pytest.fixture
+def recording(audio_source):
+    return Recording(
+        id='rec', sources=[audio_source], sampling_rate=8000, num_samples=8000, duration_seconds=1.0
+    )
+
+
+@pytest.fixture
+def cut(recording):
+    return Cut(id='cut', start=0, duration=1.0, channel=0, recording=recording)
+
+
+def test_extract_features(cut):
+    extractor = Fbank()
+    arr = cut.compute_features(extractor=extractor)
+    assert arr.shape[0] == 100
+    assert arr.shape[1] == extractor.config.num_mel_bins
```

Comment:
Is there a more generic way of querying the feature-dim of an extractor object?

Reply:
Good idea. I'll add that as a method that accepts `sampling_rate` argument so that it can be computed for `Spectrogram`.